week0003/nhantd/python/week0003.py

```diff
@@ -0,0 +1,42 @@
+import sys
+
+
+class Accounts():
+
+    def __init__(self):
+        self.data = dict()
+
+    def addAccount(self, Name, Amount):
+        if Name not in self.data and Amount >= 0:
+            self.data[Name] = Amount
+            return 'True'
+        else:
+            return 'False'
+
+    def depositMoney(self, Name, Amount):
+        if Name in self.data and Amount >= 0:
+            self.data[Name] = self.data[Name] + Amount
+            return 'True'
+        else:
+            return 'False'
+
+    def withdrawMoney(self, Name, Amount):
+        if Name in self.data and Amount >= 0 and Amount <= self.data[Name]:
+            self.data[Name] = self.data[Name] - Amount
+            return 'True'
+        else:
+            return 'False'
+
+if __name__ == "__main__":
+    sampleAccounts = Accounts()
+    for line in sys.stdin:
+        Key, Name, Amount = line.strip().split(' ')
```

Comment:
You should cut this part and put into a function to process requests. The main piece of code should only pass empty accounts & STDIN or each line. This will make your code more modular and more testable.